examples/sumo/grid.py

```diff
@@ -170,7 +170,9 @@ def grid_example(render=None, use_inflows=False):
 
     env_params = EnvParams(additional_params=ADDITIONAL_ENV_PARAMS)
 
-    tl_logic = TrafficLightParams(baseline=False)
+    tl_logic = TrafficLightParams()
+    # tl_logic.toggle_all_nodes('traffic_light')
```

Comment:
Should these be commented?

Reply:
I'd wanted to include these to show users what you can do. I can take it out though. Thoughts?